src/gui/EditWidgetIcons.cpp

```diff
@@ -219,11 +219,17 @@ void EditWidgetIcons::downloadFavicon()
     }
 
     // Try to use alternative fallback URL, if enabled
-    if (config()->get("security/IconDownloadFallback", false).toBool()) {
+    if (config()->get("security/IconDownloadFallback", false).toBool()
+        || config()->get("security/IconDownloadPrefer", false).toBool()) {
         QUrl fallbackUrl = QUrl("https://icons.duckduckgo.com");
         fallbackUrl.setPath("/ip3/" + QUrl::toPercentEncoding(fullyQualifiedDomain) + ".ico");
 
-        m_urlsToTry.append(fallbackUrl);
```

Comment:
Remove the new setting and just change this from append to prepend.

Reply:
added ut afterwards to ensure the same order and hostIsIp afterwards